### DISCUSSION TITLE
PaddleOCR/latest/ppstructure/model_train/train_table

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14270] [Q&A]
# PaddleOCR/latest/ppstructure/model_train/train_table

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/latest/ppstructure/model_train/train_table.html?h=%E8%A1%A8%E6%A0%BC

**Comment by budian** [comment]
res.save_to_img("./output/") 
res.save_to_xlsx("./output/") 

CenteOS环境，图片结果生成成功，但没有生成excel文件，也没有报错，能否帮忙解答一下，谢谢。

**Reply by budian** [reply]
1529337992@qq.com
可以发邮件联系，谢谢
